engine/graphics/src/graphics.cpp

```diff
@@ -1594,6 +1600,10 @@ namespace dmGraphics
     {
         return g_functions.m_GetNumTextureHandles(texture);
     }
+    uint32_t GetTextureUsageHintFlags(HTexture texture)
+    {
+        return g_functions.m_GetTextureUsageHintFlags(texture);
+    }
```

Comment:
I don't know exactly how yet, but it would be nice if all basic graphics types shared the same base:

```
struct Texture
{
    int m_Width;
    int m_Height;
    int m_Flags;
    ...
};

struct OpenGLTexture
{
    Texture m_Base;
    GLuint  m_Id;
    ...
};

struct VulkanTexture
{
    Texture m_Base;
    VkImage m_Image;
    ...
};
```

We could remove a TON of these Get<xxx> functions in the adapter..